test/modellib/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+add_onnx_mlir_library(ModelLib
```

Comment:
This should be excluded from the list of all libs, by default add_onnx_mlir_library will add this to the OMLibs

Reply:
I think this is done by adding this line
```
  EXCLUDE_FROM_OM_LIBS
```

first time ever editing/creating cmake files, started with a tutorial this morning :-)
Let me know if that is not what you had in mind

Reply:
Yes, `EXCLUDE_FROM_OM_LIBS` is the correct option.